.github/workflows/release.yml

```diff
@@ -0,0 +1,143 @@
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      # The checkout below is needed to use the custom actions
+      - uses: actions/checkout@v4
+      - name: Split version
+        id: split_version
+        uses: ./.github/actions/split_version
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+          release-branch: ${{ env.RELEASE_BRANCH }}
+          existing-version: ${{ env.EXISTING_VERSION }}
+      - name: Print release parameters
+        run: | 
+          echo "Releasing with the following parameters"
+          echo "Release version: ${{ env.RELEASE_VERSION }}"
+          echo "Next Development version: ${{ env.NEXT_DEVELOPMENT_VERSION }}"
+          echo "Major minor version: ${{ steps.split_version.outputs.major-minor-version }}"
+          echo "Patch Version: ${{ steps.split_version.outputs.patch-version }}"
+          echo "Is pre release version: ${{ steps.split_version.outputs.is-pre-release }}"
+          echo "Release branch: ${{ steps.split_version.outputs.release-branch }}"
+          echo "Existing version: ${{ steps.split_version.outputs.existing-version }}"
+      - name: Checkout release branch
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ steps.split_version.outputs.release-branch }}
+      - name: Update Antora Docs
+        uses: ./.github/actions/update_antora
+        with:
+          version: ${{ env.RELEASE_VERSION }}
+          major-minor-version: ${{ steps.split_version.outputs.major-minor-version }} 
+          patch-version: ${{ steps.split_version.outputs.patch-version }}
+          is-pre-release: ${{ steps.split_version.outputs.is-pre-release }}
+          release-branch: ${{ steps.split_version.outputs.release-branch }}
+          custom-facet-old-version: ${{ steps.split_version.outputs.existing-version }}
+          custom-facet-new-version: ${{ env.RELEASE_VERSION }}
+      - name: Create settings.xml
+        run: |
+          echo "<settings><servers><server><id>ossrh</id><username>${{ secrets.SONATYPE_CENTRAL_REPO_TOKEN_USER }}</username><password>${{ secrets.SONATYPE_CENTRAL_REPO_TOKEN_KEY }}</password></server><server><id>ghcr.io</id><username>${{ secrets.GHCR_IO_USERNAME }}</username><password>${{ secrets.GHCR_IO_TOKEN }}</password></server><server><id>pypi</id><username>${{ secrets.PYPI_USERNAME }}</username><password>${{ secrets.PYPI_TOKEN }}</password></server><server><id>dev-pypi</id><username>${{ secrets.TEST_PYPI_USERNAME }}</username><password>${{ secrets.TEST_PYPI_TOKEN }}</password></server> </servers></settings>" > $HOME/.m2/settings.xml
+      - name: Release aiSSEMBLE
+        run: | 
+          ./mvnw release:clean release:prepare release:perform -s $HOME/.m2/settings.xml -U -B -Pci,gh-build -DdryRun=${{ env.DRY_RUN }} -DpushChanges=true -DreleaseVersion=${{ env.RELEASE_VERSION }} -DdevelopmentVersion=${{ env.NEXT_DEVELOPMENT_VERSION }} -Dgpg.keyname=aissemble@bah.com -DsignTag=true -Dmaven.build.cache.enabled=false
```

Comment:
S: IMO the dry-run isn't going to be used enough to warrant having it checked in to dev and it just adds noise to the parameters. Esp since we aren't using it for things like the git commits at earlier steps.

Reply:
Removed